git_services/git_services/init/clone.py

```diff
@@ -11,15 +12,23 @@
 if __name__ == "__main__":
     config = config_from_env()
     setup_sentry(config.sentry)
+
+    repositories = config.repositories
+    if repositories:
+        repos = json.loads(repositories)
+        branch = repos[0]["branch"]
+        commit_sha = repos[0]["commit_sha"]
+        repository_url = repos[0]["url"]
+    else:
+        branch = config.branch
+        commit_sha = config.commit_sha
+        repository_url = config.repository_url
+
     git_cloner = GitCloner(
-        config.git_url,
-        config.repository_url,
-        config.user,
-        config.lfs_auto_fetch,
-        config.mount_path,
-    )
-    git_cloner.run(
-        session_branch=config.branch,
-        root_commit_sha=config.commit_sha,
-        storage_mounts=config.storage_mounts,
+        repositories=json.loads(config.repositories) if config.repositories else [],
+        workspace_mount_path=config.workspace_mount_path,
+        user=config.user,
+        lfs_auto_fetch=config.lfs_auto_fetch,
```

Comment:
we probably will need to set this per repository in the future

Reply:
I'll create an issue for this.